src/tracking/EventFunctions.tsx

```diff
@@ -0,0 +1,10 @@
+import { Event } from "./GoogleAnalytics";
+
+export function triggerSongView(songNumber: number, mode: String) {
+  try {
+    Event("INTERACTION", `Song viewed(${mode})`, String(songNumber));
```

Comment:
linter says use `string` not `String`. While that's true, I think what you want here is `songNumber.toString()`